search-app/src/cmr/search/services/parameters/converters/geometry.clj

```diff
@@ -35,16 +38,30 @@
   [^LineString line-string]
   (rr/ords->ring :geodetic (coords->ords (.getCoordinates line-string))))
 
+(defn force-ccw-orientation
+  "Forces a LineString to be in counter-clockwise orientation"
+  [^LineString line-string winding]
+  (let [coords (.getCoordinates line-string)]
+    (if (and 
+          (> (count coords) 3)
+          (= winding :cw))
+      (.reverse line-string)
+      line-string)))
+
 (defn polygon->shape
-  "Convert a JTS Polygon to a spatial lib shape that can be used in a Spatial query"
-  [^Polygon polygon]
-  (let [boundary-ring (.reverse (.getExteriorRing polygon))
+  "Convert a JTS Polygon to a spatial lib shape that can be used in a Spatial query.
+  The `context` map can be used to provide information about winding. Accepted keys
+  are `:boundary-winding` and `:hole-winding`. Accepted values are `:cw` and `ccw`."
```

Comment:
Should the values keywords or strings (fix either :cw or ccw)?

Reply:
Fixed